docs/int/quickstart/quickstart-mainnet.md

```diff
@@ -98,7 +98,7 @@ CHARON_BEACON_NODE_ENDPOINTS=<YOUR_REMOTE_MAINNET_BEACON_NODE_URL>
 If you are running your mainnet DV node with `mev-boost`, you need to uncomment and set the `MEVBOOST_RELAYS` variable in the `.env` file
 ```
 ...
-# MEV-Boost docker container image version, e.g. `latest` or `v1.4.0`.
+# MEV-Boost docker container image version, e.g. `latest` or `v1.5.0`.
```

Comment:
https://discord.com/channels/755466764501909692/1014218796795449404/1087341616911630478
```suggestion
# MEV-Boost docker container image version, e.g. `latest` or `1.5.0`.
```

I don't think this docker tag exists.

Reply:
Great, nice catch!